docs/v1/P4Runtime-Spec.mdk

```diff
@@ -6457,6 +6492,49 @@ applicable on the table and the APIs.
     calculation tables where matchfields are the field values and the union
     data contains the hash value.
 
+  * Indexed tables : Entries are ordered.
+
+  * Reset-only tables :
+    * `INSERT` : Not supported. Returns `NOT_SUPPORTED` error
+    * `MODIFY` : Not supported. Returns `NOT_SUPPORTED` error
+    * `DELETE` : Instead of deleting, this operation resets the entry with
+      default values for every field.
+      Only Match fields must be set. `NOT_FOUND` is returned if
+      the entry isn't found. If no Match fields are given, then all entries
+      are reset (wildcard reset). If the table contained init entries, they
+      are restored to initial state with their original union and union field
+      values.
+
+
+## GenericTable properties
+
+Properties exist at 3 different levels - Table, entry and field (match or union)
+
+### Table properties
+
+Table properties are at table level. A combination of these define a certain
+table category
+
+* Read-only
+* Modify-only
+* Reset-only
+* Indexed
+* Keyless
+* Calculation
```

Comment:
Per the meeting, some properties are mutually-exclusive and some may overlap. Could you show a matrix or truth-table showing what's possible?

Reply:
And presumably, any use of mutually exclusive properties should be rejected immediately.

Reply:
Added the matrix